android/rnkiwimobile/src/main/java/com/kiwi/rnkiwimobile/ancillaries/RNAncillaryFactoryFragment.kt

```diff
@@ -0,0 +1,9 @@
+package com.kiwi.rnkiwimobile.ancillaries
+
+import com.kiwi.rnkiwimobile.RNKiwiFragment;
+
+abstract  class RNAncillaryFactoryFragment(initialProperties: RNAncillaryFactoryInitialProperties) : RNKiwiFragment(RNAncillaryFactoryModule.getInitialProperties(initialProperties)) {
```

Comment:
Line too long...

Reply:
Is there some equivalent of eslint for this maybe?

Reply:
Same here not sure if I escaped to new line properly.